config/params.php

```diff
@@ -68,17 +62,9 @@
         ],
     ],
 
-    'app' => [
-        'charset' => 'UTF-8',
-        'language' => 'en',
-        'name' => 'My Project',
-    ],
-
-    'mailer' => [
-        'adminEmail' => 'admin@example.com',
-    ],
-
     'yiisoft/router' => [
         'enableCache' => false,
     ],
+
+    ReverseBlockMerge::class => new ReverseBlockMerge(),
```

Comment:
Why do we need it?

Reply:
That the parameters are merged in order.